src/dicom-parse.c

```diff
@@ -954,16 +954,11 @@ bool dcm_parse_pixeldata_offsets(DcmError **error,
         // the BOT is missing, we must scan pixeldata to find the position of
         // each frame
 
-        // we could use our generic parser above ^^ but we have a special loop
-        // here as an optimisation (we can skip over the pixel data itself)
-
         dcm_log_info("building Offset Table from Pixel Data");
 
         // 0 in the BOT is the offset to the start of frame 1, ie. here
         *first_frame_offset = position;
-
-        position = 0;
-        for (int i = 0; i < num_frames; i++) {
+	for (int i = 0; i < num_frames; i++) {
```

Comment:
This indent looks bad. I think you have your editor set for tab == 8 spaces, but it should be 4 for this project. It's probably best to set your editor to expand tabs to spaces.

There's a `.editorconfig` in the project root, maybe you saw. Perhaps you are using something that does not support per-project configs?

https://github.com/ImagingDataCommons/libdicom/blob/main/.editorconfig